XcodeLegacy.sh

```diff
@@ -1155,7 +1153,7 @@ SPEC_EOF
                 fi
             done
         fi
-        if [ -f "$GCCDIR/Platforms/MacOSX.platform/Info.plist-original" ]; then
+        if [ -f "$GCCDIR/Platforms/MacOSX.platform/Info.plist-original" ] && [ $(ls -1 "$SDKDIR"/SDKs/MacOSX*.sdk/legacy 2>/dev/null | wc -l) -eq 0 ]; then
```

Comment:
wow this is ugly, but let's keep this!

Reply:
Yeah, it was the best compact solution I could find for checking that an arbitrary number of file locations do not exist.